deploy/helm/sumologic/conf/setup/custom.sh

```diff
@@ -36,5 +36,10 @@ for dir in $(ls -1 /customer-scripts | grep _ | grep -oE '^.*?_' | sed 's/_//g'
   for file in $(ls -1 "/customer-scripts/${dir}_"* | grep -oE '_.*' | sed 's/_//g'); do
     cp "/customer-scripts/${dir}_${file}" "${target}/${file}"
   done
+
+  [[ ! -f setup.sh ]] \
+    && echo "You're missing setup.sh script in custom scripts directory: '${dir}'" \
+    && continue
```

Comment:
I think `if then` would be better to use here than oneliner

Reply:
Please recheck